src/App.js

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { fetchRockets } from './redux/rockets/rocketsSlice';
+import { fetchMissions } from './redux/missions/missionsSlice';
+import Layout from './routes/Layout';
+import Profile from './routes/Profile';
+import Rockets from './routes/Rockets';
+import Missions from './routes/Missions';
+import NotMatch from './routes/NotMatch';
+import './App.css';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+    dispatch(fetchMissions());
```

Comment:
- [ ] The missions should not be fetched when we load the app in the browser. They should only be fetched when we navigate to the missions page. Kindly fix that.

Reply:
thanks for pointing it out.

Reply:
solved